### PR TITLE
Add response_content_disposition option

### PR DESCRIPTION
**Pull request by yosangwon**
Example:

```ruby
require "gcs-signer"

signer = GcsSigner.new
signer.sign_url(
  "test-bucket",
  "hello.txt",
  response_content_disposition: "attachment; filename=world.txt"
)
```

### CODE REVIEW

File: lib/gcs_signer.rb [hunk 1]
```diff
@@ -114,9 +114,14 @@ def string_that_will_be_signed(url, options)
 
   # Escapes and generates query string for actual result.
   def query_for_signed_url(signature, options)
-    URI.encode_www_form GoogleAccessId: options[:google_access_id],
-                        Expires: options[:expires].to_i,
-                        Signature: Base64.strict_encode64(signature)
+    query = {
+      "GoogleAccessId" => options[:google_access_id],
+      "Expires" => options[:expires].to_i,
```
Comment (premist) [inline lib/gcs_signer.rb:128]: Do we need to perform this step instead expecting user to do this before calling this method?

File: lib/gcs_signer.rb [hunk 2]
```diff
@@ -114,9 +114,14 @@ def string_that_will_be_signed(url, options)
...
+      "Signature" => Base64.strict_encode64(signature),
+      "response-content-disposition" => options[:response_content_disposition]
+    }.reject { |_, v| v.nil? }
+
+    URI.encode_www_form query
```
Comment (premist) [inline lib/gcs_signer.rb]: I'm not sure here: Do we often use method call without parenthesis in this project?

File: lib/gcs_signer.rb (hunk 1 above)
Comment (yosangwon) [reply lib/gcs_signer.rb:128]: Yes. To allow DateTime objects other date-time sugars to be `valid_for` and `expires`. See [documentations](http://www.rubydoc.info/gems/gcs-signer/0.2.1/GcsSigner:sign_url)

File: lib/gcs_signer.rb (hunk 2 above)
Comment (yosangwon) [reply lib/gcs_signer.rb]: Done!

**Pull request merged**